src/types.ts

```diff
@@ -20,12 +20,21 @@ export type GenerateURL = <T = any>(
   args: ContextType & { doc: T; locale?: string },
 ) => string | Promise<string>
 
+interface FieldOptions {
+  title: boolean
+  description: boolean
+  image: boolean
+}
+
 export interface PluginConfig {
   collections?: string[]
   globals?: string[]
   uploadsCollection?: string
   fields?: Field[]
   tabbedUI?: boolean
+  required?: FieldOptions
+  localized?: FieldOptions
+  interfaceName?: string
```

Comment:
Problem for a different day, but we should probably pull types like this from `payload` where possible.

Reply:
How is that done?

Reply:
Gave it a shot - see latest commit